packages/web-components/src/global/variables.css

```diff
@@ -279,7 +279,9 @@
   --ic-space-xxl: 3rem;
 
   /* z-index */
+  --ic-z-index-base-value: 1000;
```

Comment:
Is this for testing or meant to overwrite the previous z-index ticket? https://github.com/mi6/ic-ui-kit/pull/471

Reply:
now #471 is merged, can rebase & use the new properties/values